src/hardware/serialport/softmodem.cpp

```diff
@@ -150,6 +150,15 @@ CSerialModem::CSerialModem(const uint8_t port_idx, CommandLine *cmd)
 		        GetPortNumber(), telnet_mode ? "enabled" : "disabled");
 	}
 
+	// Get the connect speed to report
+	if (getUintFromString("connectspeed:", val, cmd)) {
+		LOG_MSG("SERIAL: Port %" PRIu8 " will report connect speed %d",
+			GetPortNumber(), val);
+	} else {
+		val = 57600;
+	}
+	snprintf(connect_string, 255, "CONNECT %d", val);
```

Comment:
Suggest using `baudrate` as the setting name, as it was the common term to describe the modem's transfer rate or speed.

For example, google the terms "modem connect speed" vs "modem baud rate", and there are almost no hits for the prior but lots of authoritative direct hits for the latter:

#### Linux documentation project

![2022-07-09_17-56_1](https://user-images.githubusercontent.com/1557255/178127320-b3baf18c-0a13-409b-a95c-3b03746ace96.png)

#### IBM documentation

![2022-07-09_17-56](https://user-images.githubusercontent.com/1557255/178127321-25199eff-ef6e-44ba-98ed-160e0bddb2a4.png)